podaac/l2ss.py

```diff
@@ -0,0 +1,41 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Hi @Omkar20895 can you please change the file header to the following
```
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```
We are not licensing this code ```to``` the Apache Software Foundation, instead we are licensing it under the Apache Software License,=. Thank you so much. Please do this with all of the files for Podaacpy from now on. Thank you @Omkar20895